src/pages/Account/Account.jsx

```diff
@@ -16,6 +17,7 @@ const Account = ({openConfirmModal, completedTasks, accountData, logout }) => {
         <CompletedTask text={text} key={index} />
       ))}
       <RedButton text="1 month of service" text2="5 FOOBAR" className="pay-button" handleOnClick={openConfirmModal} />
+      <h4 className="error">{error ? error : ''}</h4>
```

Comment:
Let's conditionally render the entire `h4` tag so that we don't have an empty `h4` when there is no `error` message.

```js
{error ? <h4>...</h4> : null}
```